py/kubeflow/kfctl/testing/pytests/endpoint_ready_test.py

```diff
@@ -15,7 +15,7 @@
 
 # There's really no good reason to run test_endpoint during presubmits.
 # We shouldn't need it to feel confident that kfctl is working.
-@pytest.mark.skipif(os.getenv("JOB_TYPE") == "presubmit",
+@pytest.mark.skipif(os.getenv("JOB_TYPE") == "periodic",
```

Comment:
Why change this?

Reply:
to enable the test - will rebase once the fix is confirmed.

Reply:
@gabrielwen nit might be easier to just comment out the  @pytest.... rather than changing  the value to periodic. This way even if the code gets merged we wouldn't disable it in the periodic tests